metricbeat/beat/list.go

```diff
@@ -0,0 +1,25 @@
+package beat
+
+// Make sure all active plugins are loaded
+// TODO: create a script to automatically generate this list
+import (
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/helper"
+
+	// List of all metrics to make sure they are registred
+	// Every new metric must be added here
+	_ "github.com/elastic/beats/metricbeat/module/beat"
+	_ "github.com/elastic/beats/metricbeat/module/elasticsearch/cluster-stats"
+	_ "github.com/elastic/beats/metricbeat/module/elasticsearch/info"
+	_ "github.com/elastic/beats/metricbeat/module/elasticsearch/stats"
+	_ "github.com/elastic/beats/metricbeat/module/redis/info"
```

Comment:
This would definitively be nicer as this list here would have to be updated less frequently and it imports would be more local. I tried this first but it doesn't work because of cyclic imports. As I designed metricbeat so far that a MetricSet registers with the module and the module doesn't know in advance about all the MetricSets, the MetricSet imports its module. Not sure if there is a way around the cyclic imports to make it mor maintainable. An other option could be to automatically generate this list.